### PR TITLE
Updating common file to contain active URLs

### LINKED ISSUE #769: SES7: 1  Available Documentation hyperlinks are reversed
### Type of report
   - [x] bug
   - [ ] feature request

### Source Document
1  Available Documentation:

https://documentation.suse.com/ses/7/single-html/ses-deployment/#id-1.3.2.7

### Summary

This section presents a list of documentation resources (books): 

* Deployment Guide
* Book “Operations and Administration Guide”
* Book “Security Hardening Guide”
* Book “Troubleshooting Guide”
* Book “SUSE Enterprise Storage for Windows Guide”

The first item of the list is the Deployment Guide itself (the very book we are reading). This one is a hyperlink, and the remaining ones (Book "X", Book "Y", etc.) are not hyperlinks.

It should be the opposite. The reader does not need a hyperlink to the very book they are reading. They need hyperlinks to the *other* books.

### PR DESCRIPTION
**Pull request by asettle**
Closes #769

### CODE REVIEW

File: xml/common_intro_available_doc.xml [hunk 1]
```diff
@@ -46,7 +46,8 @@
 
  <variablelist>
   <varlistentry>
-   <term><xref linkend="book-storage-deployment"/></term>
+   <link xlink:href="https://documentation.suse.com/ses/7/html/ses-all/book-storage-deployment.html">
+    <citetitle>Deployment Guide</citetitle></link>
```
Comment (tomschr) [inline xml/common_intro_available_doc.xml:50]: Try to add the `citetitle` without a linebreak:

```suggestion
   <link xlink:href="https://documentation.suse.com/ses/7/html/ses-all/book-storage-deployment.html"
    ><citetitle>Deployment Guide</citetitle></link>
```

File: xml/common_intro_available_doc.xml [hunk 2]
```diff
@@ -56,7 +57,8 @@
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term><xref linkend="book-storage-admin"/></term>
+   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-admin.html">
+    <citetitle>Administration and Operations Guide</citetitle></link>
```
Comment (tomschr) [inline xml/common_intro_available_doc.xml:61]: Same here:

```suggestion
   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-admin.html"
    ><citetitle>Administration and Operations Guide</citetitle></link>
```

File: xml/common_intro_available_doc.xml [hunk 3]
```diff
@@ -67,15 +69,17 @@
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term><xref linkend="book-storage-security"/></term>
+   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-security.html">
+    <citetitle>Security Hardening Guide</citetitle></link>
```
Comment (tomschr) [inline xml/common_intro_available_doc.xml:73]:
```suggestion
   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-security.html"
    ><citetitle>Security Hardening Guide</citetitle></link>
```

File: xml/common_intro_available_doc.xml [hunk 4]
```diff
@@ -67,15 +69,17 @@
...
    <listitem>
     <para>
      This guide focuses on how to ensure your cluster is secure.
     </para>
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term><xref linkend="book-storage-troubleshooting"/></term>
+   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-troubleshooting.html">
+    <citetitle>Troubleshooting Guide</citetitle></link>
```
Comment (tomschr) [inline xml/common_intro_available_doc.xml:82]:
```suggestion
   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-troubleshooting.html"
    ><citetitle>Troubleshooting Guide</citetitle></link>
```

File: xml/common_intro_available_doc.xml [hunk 5]
```diff
@@ -85,7 +89,8 @@
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term><xref linkend="book-storage-windows"/></term>
+   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-windows.html">
+    <citetitle>&productname; for Windows Guide</citetitle></link>
```
Comment (tomschr) [inline xml/common_intro_available_doc.xml:93]:
```suggestion
   <link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-windows.html"
    ><citetitle>&productname; for Windows Guide</citetitle></link>
```

**Review by tomschr — CHANGES_REQUESTED** [review]
Just minor things with linebreaks. Other than that it looks good. :+1:

**Comment by asettle** [comment]
Thanks @tomschr somehow I missed those breaks in my editor :)

**Comment by tomschr** [comment]
No problem. It could (potentially) lead to spurious whitespaces in the link (which we should avoid).

**Review by lproven — APPROVED** [review]

File: xml/common_intro_available_doc.xml [hunk 6]
```diff
@@ -85,7 +89,8 @@
    </listitem>
   </varlistentry>
   <varlistentry>
-   <term><xref linkend="book-storage-windows"/></term>
+   <term><link xlink:href="https://documentation.suse.com/ses/html/ses-all/book-storage-windows.html"
+    ><citetitle>&productname; for Windows Guide</citetitle></link></term>
    <listitem>
     <para>
      This guide describes the integration, installation and configuration of
```
Comment (tbazant) [inline xml/common_intro_available_doc.xml:96]:
```suggestion
     This guide describes the integration, installation, and configuration of
```

**Review by tbazant — APPROVED** [review]
looks good

**Pull request merged**